File: Web.Admin/ClientApp/src/app/models/person/PersonNote.ts
export class PersonNote {

  id: number;
  subject: string;
  body: string;
  time: string;

  public PersonNote(id: number,
      subject: string,
      body: string,
      time: string) {
    this.id = id;
    this.subject = subject;
    this.body = body;
    this.time = time;
  };

}

File: Libraries/Shared/NIST.CVP.Libraries.Shared.ACVPCore.Abstractions/Models/PersonNote.cs
using System;

namespace NIST.CVP.Libraries.Shared.ACVPCore.Abstractions.Models
{
    public class PersonNote
    {
        public long ID { get; set; }
        public string Subject { get; set; }
        public string Body { get; set; }
        public DateTime? Time { get; set; }
    }
}
